src/algebra/symmetrized.lean

```diff
@@ -141,44 +141,37 @@ instance [add_comm_group α] : add_comm_group (αˢʸᵐ) :=
 { ..sym_alg.add_comm_monoid, ..sym_alg.add_group }
 
 instance {R : Type*} [semiring R] [add_comm_monoid α] [module R α] : module R αˢʸᵐ :=
-function.injective.module R ⟨unsym, rfl, λ _ _, rfl⟩ (λ _ _, id) (λ _ _, rfl)
+function.injective.module R ⟨unsym, unsym_zero, unsym_add⟩ unsym_injective unsym_smul
 
 lemma mul_def [ring α] [invertible (2 : α)] (a b : αˢʸᵐ) :
   a * b = sym (⅟2*(unsym a * unsym b + unsym b * unsym a)) := by refl
 
+lemma unsym_mul [has_mul α] [has_add α] [has_one α] [invertible (2 : α)] (a b : αˢʸᵐ) :
+  unsym (a * b) = ⅟2*(unsym a * unsym b + unsym b * unsym a) := by refl
+
+lemma sym_mul_sym [has_mul α] [has_add α] [has_one α] [invertible (2 : α)] (a b : α) :
+  sym a * sym b = sym (⅟2*(a * b + b * a)) :=
+rfl
+
 instance [has_mul α] [has_add α] [has_one α] [invertible (2 : α)] (a : α) [invertible a] :
   invertible (sym a) :=
-{ inv_of := sym ⅟a,
+{ inv_of := sym (⅟a),
```

Comment:
@mans0954, note that what you had before was `(sym inv_of) a` (nonsense!), which is why you needed the `change`s

Reply:
Thanks, again I'm surprised that Lean allowed me to write nonsense without complaining.